src/py_scripts/fc_fetch_reads.py

```diff
@@ -0,0 +1,3 @@
+from falcon_kit.mains.fetch_reads import main
```

Comment:
The old programs are now *also* in `setup.py`. The wrappers in `src/py_scripts` are deprecated, for a not-very-good-reason: When the scripts are run via `main()` in the package tree, and if they have proper `if __name__ == "__main__"` wrapping, then they can also be executed like this:

    python -m 'falcon_kit.mains.run_prog'

And *that* helps us because then we don't have to register the new executables in every g*d-d@mn installation script that Herb dreams up. That's how we want to run these scripts from bash too. I'll take care of it.